src/infrastructure/jira/inbound-auth/jira-context-symmetric-jwt-token-verifier.ts

```diff
@@ -7,10 +7,9 @@ import {
 import { verifyExpClaim } from './jira-jwt-utils';
 
 import { isEnumValueOf } from '../../../common/enumUtils';
+import type { JSONSchemaTypeWithId } from '../../../common/schema-validation';
+import { assertSchema } from '../../../common/schema-validation';
 import type { ConnectInstallation } from '../../../domain/entities';
-import { UnauthorizedError } from '../../../web/middleware/errors';
-import type { JSONSchemaTypeWithId } from '../../index';
-import { assertSchema, getLogger } from '../../index';
 import { connectInstallationRepository } from '../../repositories';
 
 type JiraContextSymmetricJwtClaims = {
```

Comment:
**note:** While diff looks a quite significant, there is no logical change in this file. The PR:

- Removes `try/catch` to let errors propagate further. 
- Throws `Error` instead of a custom error. *All errors* thrown from this module are mapped to `UnauthorizedResponseStatusError` in the corresponding middleware to avoid exposing auth-related details to a user. If we need at some point a more granular error handling, we can introduce additional errors specific to this module.